src/applications/disability-benefits/686c-674/containers/App.jsx

```diff
@@ -27,7 +27,7 @@ function App({ location, children, isLoggedIn, isLoading, vaFileNumber }) {
   // redirect them to the introduction page.
   if (
     !isLoggedIn ||
-    (isLoggedIn && !vaFileNumber?.hasVaFileNumber?.validVAFileNumber)
+    (isLoggedIn && !vaFileNumber?.hasVaFileNumber?.VALIDVAFILENUMBER)
```

Comment:
I don't see that vets-api PR making this all caps... but okay...

Reply:
Maybe it should be `ValidVaFileNumber`?

Reply:
hey @Mottie I'm converting to all caps [here](https://github.com/department-of-veterans-affairs/vets-website/pull/13941/files#diff-c31648113abbbff1c23a1debd7cb287dR31). The issue is with the variance in inflections vets-api is having.